### ISSUE TITLE
Kubo/core is borken 

### ISSUE THREAD

**Issue by AxelGard** [ipfs/kubo#9852]
### Checklist

- [X] This is a bug report, not a question. Ask questions on [discuss.ipfs.io](https://discuss.ipfs.io).
- [X] I have searched on the [issue tracker](https://github.com/ipfs/kubo/issues?q=is%3Aissue) for my bug.
- [X] I am running the latest [kubo version](https://dist.ipfs.tech/#kubo) or have an issue updating.

### Installation method

ipfs-desktop

### Version

```Text
v0.19.2
```

### Config

```json
Go mod file

go 1.20

require (
	github.com/ipfs/go-cid v0.4.1
	github.com/ipfs/kubo v0.19.2
	github.com/libp2p/go-libp2p v0.27.1
	github.com/libp2p/go-libp2p-kad-dht v0.23.0
	github.com/multiformats/go-multiaddr v0.9.0
)


```

### Description

I am working with kubo using go lang, 
more specifically kubo/core. 

If a use the core package in any way, for example:
```go
package main

import (
	"fmt"

	core "github.com/ipfs/kubo/core"
)

func main() {
	conf := core.BuildCfg{}
	fmt.Println(conf)
}
```
I get the error 
```
# github.com/ipfs/go-libipfs/bitswap/client
/home/axel/go/pkg/mod/github.com/ipfs/go-libipfs@v0.6.2/bitswap/client/client.go:138:10: cannot use bssession.New(sessctx, sessmgr, id, spm, pqm, sim, pm, bpm, notif, provSearchDelay, rebroadcastDelay, self) (value of type *session.Session) as sessionmanager.Session value in return statement: *session.Session does not implement sessionmanager.Session (wrong type for method GetBlock)
                have GetBlock(context.Context, cid.Cid) ("github.com/ipfs/go-libipfs/blocks".Block, error)
                want GetBlock(context.Context, cid.Cid) ("github.com/ipfs/go-block-format".Block, error)
/home/axel/go/pkg/mod/github.com/ipfs/go-libipfs@v0.6.2/bitswap/client/client.go:259:9: cannot use session.GetBlocks(ctx, keys) (value of type <-chan "github.com/ipfs/go-block-format".Block) as <-chan "github.com/ipfs/go-libipfs/blocks".Block value in return statement
# github.com/ipfs/kubo/thirdparty/verifbs
/home/axel/go/pkg/mod/github.com/ipfs/kubo@v0.19.2/thirdparty/verifbs/verifbs.go:29:38: cannot use blks (variable of type []"github.com/ipfs/go-libipfs/blocks".Block) as []"github.com/ipfs/go-block-format".Block value in argument to bs.GCBlockstore.PutMany
/home/axel/go/pkg/mod/github.com/ipfs/kubo@v0.19.2/thirdparty/verifbs/verifbs.go:56:36: cannot use blks (variable of type []"github.com/ipfs/go-libipfs/blocks".Block) as []"github.com/ipfs/go-block-format".Block value in argument to bs.Blockstore.PutMany
```

if I do 

```go
package main

import (
	"fmt"

	kubo "github.com/ipfs/kubo"
)

func main() {
	fmt.Println(kubo.GetUserAgentVersion())
}

```

then that works fine and I get: `kubo/0.19.2/`

What is wrong here? 
I hade code that worked before now kubo/core does not work at all? Everything just gets this error! 
I tried updating, but the same error is there.

**Comment by AxelGard** [comment]
Fixed markdown error and removed indirect imports

**Comment by guseggert** [comment]
Thanks for the report! This should be fixed in Kubo v0.20, which will be released in a few days. In the meantime you can unblock your build by using kubo v0.20.0-rc2 (with the usualy caveats of RC releases).

This is related to "github.com/ipfs/go-block-format" being inadvertently moved into go-libipfs...we have since removed it which fixes this issue.

Sorry for the inconvenience :(

**Comment by AxelGard** [comment]
Okay thanks @guseggert
